<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN216042" failures="2" tests="2" name="Test" time="105.362" errors="0" timestamp="2024-03-18T11:04:27 IST">
  <testcase classname="testCases.TC_002_LoginTest" name="verify_login" time="39.687">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_002_LoginTest.verify_login(TC_002_LoginTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="verify_account_registration" time="44.953">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_001_AccountRegistrationTest.verify_account_registration(TC_001_AccountRegistrationTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_registration -->
</testsuite> <!-- Test -->
